staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -246,7 +259,42 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 	return errs[0]
 }
 
+// whether the namespace is exempted from the webhook.
+func (a *GenericAdmissionWebhook) exemptNamespace(h *v1alpha1.ExternalAdmissionHook, attr admission.Attributes) (bool, error) {
```

Comment:
Doesn't this fail when it restricts namespaces and I'm trying to create a namespace?

Reply:
Good point. I updated the code to use the labels of the namespace object itself if the `attr.Object` is a namespace.